content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -506,30 +506,32 @@ PersistentVolume types are implemented as plugins. Kubernetes currently supports
   mounted on nodes.
 * [`nfs`](/docs/concepts/storage/volumes/#nfs) - Network File System (NFS) storage
 
-The following types of PersistentVolume are deprecated.
-This means that support is still available but will be removed in a future Kubernetes release.
+The following types of PersistentVolume are deprecated but still available.
+If you are using these volume types expect `flexVolume`, `cephfs` and `rbd`, 
+please install corresponding CSI drivers.
 
+* [`awsElasticBlockStore`](/docs/concepts/storage/volumes/#awselasticblockstore) - AWS Elastic Block Store (EBS)
+  (**migration on by default** starting v1.23) 
```

Comment:
Should we also add when CSI migration for EBS moved to GA?  Similar comment for other plugins.

Reply:
I think that the migration timeline should be included in the `content/en/docs/concepts/storage/volumes.md` page. It is unnecessary to repeat the information in the `content/en/docs/concepts/storage/persistent-volumes.md` page. In this section, it's enough for users to know which version of Kubernetes is starting to enable the migration feature by default. what do you think?